src/core/definitions.jl

```diff
@@ -20,26 +20,20 @@ const JuMPVariableArray = JuMP.Containers.DenseAxisArray{JuMP.VariableRef}
 const JuMPParamArray = JuMP.Containers.DenseAxisArray{PJ.ParameterRef}
 const DenseAxisArrayContainer = Dict{Symbol, JuMP.Containers.DenseAxisArray}
 
-@enum BUILD_STATUS begin
-    BUILT = 0
+IS.@scoped_enum BUILD_STATUS begin
     IN_PROGRESS = -1
+    BUILT = 0
     FAILED_BUILD = 1
     EMPTY = 2
 end
 
-@enum RUN_STATUS begin
+IS.@scoped_enum RUN_STATUS begin
     READY = -1
     SUCCESSFUL_RUN = 0
```

Comment:
The "RUN" is redundant. You could have READY, SUCCESSFUL, RUNNING, FAILED.

Reply:
addressed in c6af5b0